docs/github-connector/installation.md

```diff
@@ -1,9 +1,67 @@
+
+> **OPTIONAL:** Follow these steps to install the default application. Be aware that it has **full permissions** in the repository/organization.
+>
+> 1. Go to the [authentication page](https://auth-github-connector.herokuapp.com/). Click the **GitHub** button, which redirects you to another page. Select the repositories or organizations you want to install the application to and click **Install**.
+>       - **NOTE:** If the link does not work, see [this tutorial](https://developer.github.com/apps/quickstart-guides/setting-up-your-development-environment/#step-2-register-a-new-github-app) in the GitHub documentation to create your own application.
+> 2. Copy the Authentication Token. You will need it later in the installation process.
+
+### Steps
+
+1. In Kyma console, access the **Add-Ons Config** menu.
+2. Click **Add New Configuration** and in fill in the **Urls\*** field with this URL:
+
+   ```http
+   github.com/kyma-incubator/hack-showcase//addons
+   ```
+
+3. Go to the Namespace in which you install the Connector.
+4. Find the Add-On in the Service Catalog and click it.
+5. Click **Add** and select the installation plan. Fill in all required fields and click **Create Instance**.
+6. Go to the **Services** tab in the Service Catalog. After provisioning and automatic registration of application's resources, the Service Class of GitHub Connector appears here.
+7. Click the Service Class to enter its specification screen, click **Add once**, and then **Create Instance**.
+
+After you have created the service, you can easily bind it to the Lambda Function to allow the use of GitHub Events.
+
+### Verification
+
+- To verify correct configuration, check if Add-Ons and Service instances in the **Instances** area of the Service Catalog have status **RUNNING**.
+- Access **Webhooks**  in your GitHub repository or organization's **Settings** and verify that the webhook is **Active**.
+
+### Removal
+
+To correctly remove all GitHub Connector resources, you must delete them in order reverse to the installation steps.
+> **CAUTION:** Wait until deprovisioning and removing of all elements is complete before proceeding to next step to avoid possible errors. For example, after removing ServiceClass, the removal of ServiceInstance is impossible.
+
+1. Delete all service bindings from Lambda Functions and other bindings connected with your GitHub Connector Service Instance.
+2. Delete the GitHub Connector Service Instance found under the **Services** tab in the **Instances** area.
+3. Delete the GitHub Connector Add-On Instance found in the **Add-Ons** tab.
+4. To remove the Add-On Configuration, find it in the global **Add-Ons Config** catalog and remove it.
+   > **CAUTION**: This step also removes the Slack Connector template.
```

Comment:
Maybe add a new line before 
> **CAUTION** 

?

Reply:
I don't think markdown understands to put that on the same indentation level :/